src/pharmpy/tools/retries/tool.py

```diff
@@ -0,0 +1,274 @@
+import random
+import warnings
+from dataclasses import dataclass, replace
+from typing import Optional
+
+from pharmpy.internals.fn.signature import with_same_arguments_as
+from pharmpy.internals.fn.type import with_runtime_arguments_type_check
+from pharmpy.model import Model
+from pharmpy.modeling import calculate_parameters_from_ucp, calculate_ucp_scale, update_inits
+from pharmpy.tools import summarize_modelfit_results
+from pharmpy.tools.common import ToolResults, create_results
+from pharmpy.tools.modelfit import create_fit_workflow
+from pharmpy.workflows import ModelEntry, Task, Workflow, WorkflowBuilder, call_workflow
+from pharmpy.workflows.results import ModelfitResults
+
+SCALES = frozenset(('normal', 'UCP'))
+
+
+@dataclass
+class Retry:
+    modelentry: ModelEntry
+    number_of_retries: int
+
+
+def create_workflow(
+    model: Optional[Model] = None,
+    results: Optional[ModelfitResults] = None,
+    number_of_candidates: int = 5,
+    allowed_variance: float = 0.1,
+    strictness: Optional[str] = "minimization_successful or (rounding_errors and sigdigs >= 0.1)",
+    scale: Optional[str] = "UCP",
+    prefix_name: Optional[str] = "",  # FIXME : Remove once new database has been implemented
+):
+    """
+    Run retries tool.
+
+    Parameters
+    ----------
+    model : Optional[Model], optional
+        Model object to run retries on. The default is None.
+    results : Optional[ModelfitResults], optional
+        Connected ModelfitResults object. The default is None.
+    number_of_candidates : int, optional
+        Number of retry candidates to run. The default is 5.
+    allowed_variance: float
```

Comment:
The name of `allowed_variance`  should be changed. It is not a variance and allowed is kind of implicit. I have tried to find a good word for this, but I only have some suggestions. `degree` is what PsN is using. Otherwise `range` or `interval`.

Reply:
I see your point. As range and interval (for me at least) represent the possible numbers rather than the percentage of allowed increase/decrease, I decided to change it to `degree`. I had a hard time coming up with other alternatives, but please let me know if some other word comes to mind.

Reply:
I think degree is fine. I couldn't find a better word either.